mail/dovecot/Portfile

```diff
@@ -153,7 +221,7 @@ if {${name} eq ${subport}} {
     }
 
     variant solr requires lucene description {Enable apache-solr support} {
-        depends_lib-append          port:curl port:expat
+        depends_lib-append          port:clucene port:curl port:expat
```

Comment:
Requiring the lucene variant makes this dependency redundant.

Reply:
Whoops I'll replace this.